improver/cli/spot_extract.py

```diff
@@ -195,13 +195,13 @@ def process(
                 raise ValueError(msg)
 
     # Check whether a lapse rate cube has been provided
-    if apply_lapse_rate_correction and lapse_rate:
-        plugin = SpotLapseRateAdjust(
-            neighbour_selection_method=neighbour_selection_method
-        )
-        result = plugin(result, neighbour_cube, lapse_rate)
-    elif apply_lapse_rate_correction and not lapse_rate:
-        if not suppress_warnings:
+    if apply_lapse_rate_correction:
+        if len(cubes) == 3:
```

Comment:
Was a constrained load based on metadata not possible in identifying `lapse_rate` etc. rather than relying on cube ordering?

Reply:
Possibly, but it wouldn't fit the pattern for other CLIs. It is something we should look at in the future though